bash/code_to_gast/bash_helpers.js

```diff
@@ -0,0 +1,46 @@
+const bash_router = require("./bash_router.js")
+
+
+function script_to_gast(node) {
+    gast = {"type": "root"}
+    gast["body"] =  bash_router.node_to_gast(node["commands"])
+    return gast
+}
+
+function array_to_gast(node) {
+    let node_list = []
+    for(const elem of node) {
+        node_list.push(bash_router.node_to_gast(elem))
+    }
+    return node_list
+}
+
+function word_to_gast(node) {
+    text = node["text"]
+    if (text === "true") {
+        return {"type": "bool", "value": 1}
+    } else if (text === "false") {
+        return {"type": "bool", "value": 0}
+    } else if (!isNaN(text)) {
+        return {"type": "num", "value": Number(text)}
+    } else {
+        return {"type": "str", "value": text}
+    }
+}
+
+function command_to_gast(node) {
```

Comment:
Good way of dealing with this issue! I think because bash is so weird, you unfortunately probably should add some more comments / examples / documentation.